lib/cucumber/formatter/ansicolor.rb

```diff
@@ -82,54 +108,16 @@ module ANSIColor
       ALIASES.each_key do |method_name|
         next if method_name =~ /.*_param/
 
-        code = <<-COLOR
-          def #{method_name}(string=nil, &proc)
-            #{"#{ALIASES[method_name].split(',').join('(')}(string, &proc#{')' * ALIASES[method_name].split(',').length}"}
```

Comment:
Is the method signature copied anywhere or have we removed this?

Reply:
Sorry, I don't get your question :(

Reply:
I couldn't find this method signature, i.e. the split(',').join('(') e.t.c.

Reply:
I have rewritten that as part.

I had a hard time figuring out what was actually doing `.split.join * .split.lengh`.

I hope the new implementation is easier to understand!?